src/widget/form/settings/generalform.cpp

```diff
@@ -84,8 +84,6 @@ static QStringList langs = {"Български",
                             "简体中文"};
 
 static QStringList timeFormats = {"hh:mm AP", "hh:mm", "hh:mm:ss AP", "hh:mm:ss"};
```

Comment:
Shouldn't second and last one have `HH` instead of `hh` ?
![date](https://cloud.githubusercontent.com/assets/3148759/12373095/1ab29b38-bc66-11e5-9aac-943388f0003e.png)